server/migrator/MigratorClient.java

```diff
@@ -57,6 +57,23 @@ public boolean importData(String database, String filename, Map<String, String>
         return streamObserver.success();
     }
 
+    public boolean exportData(String database, String filename) {
+        MigratorProto.ExportData.Req req = MigratorProto.ExportData.Req.newBuilder()
+                .setDatabase(database)
+                .setFilename(filename)
+                .build();
+        CountDownLatch latch = new CountDownLatch(1);
+        ProgressPrinter progressPrinter = new ProgressPrinter("export");
+        ResponseObserver streamObserver = new ResponseObserver(progressPrinter, latch);
+        stub.exportData(req, streamObserver);
+        try {
+            latch.await();
```

Comment:
I know the `ResponseObserver` isn't currently touched in this PR but is there any reason for the `CountDownLatch` to be initialized _outside_ the Observer?

I think it would be cleaner if the `CountDownLatch` was initialized inside the constructor of `ResponseObserver`, and `ResponseObserver` provided its own `await()` method that simply called `latch.await()`.

Reply:
Could do that.